src/components/ble/DfuService.h

```diff
@@ -9,13 +9,13 @@
 #undef max
 #undef min
 
+#include <drivers/SpiNorFlash.h>
+#include "infinitime.h"
```

Comment:
later this header is included as `port/infinitime.h` I think this was unintentionally left out here

```suggestion
#include "port/nfinitime.h"
```